### DISCUSSION TITLE
Taiko with Firefox browser - Error: The Browser instance was closed either via `closeBrowser()` call, or it exited for reasons unknown to Taiko. You can try launching a fresh instance using `openBrowser()` or inspect the logs for details of the possible crash.

### DISCUSSION THREAD

**Discussion by karishmayadava23** [getgauge/taiko#2697] [Development]
Hi Team,

Note - I did set the system env variable to executable firefox file path.
While launching taiko with Firefox getting below issues
1. npx taiko scriptfile.js - Launching the firefox with below error and terminating the execution
![image](https://user-images.githubusercontent.com/59534066/231203922-88c85cdb-67cc-493f-80ca-19edbb22e60f.png)

2. npx taiko - launching browser , and displaying the same error but not getting out of the repl mode. For performed action displaying no logs on terminal such that navigated to url . when gave closeBrowser() terminal displays browser is closed but it is  not
![image](https://user-images.githubusercontent.com/59534066/231204661-3fc7b7e9-a85b-4827-97f5-5fa887db6fcc.png)

**Comment by DCoomer** [comment]
what does your  scriptfile.js look like?

Did you use await openBrowser() ?
